.github/pull_request_template.md

```diff
@@ -20,7 +20,10 @@
 
 #### Testing checklist
 
-Replace this dummy text with the testing checklist from the issue this pull request fixes (use checkboxes similar to how it's done in the **General checklist** above).
+Note (to contributor taking on this task assignment): Add the testing checklist below (without this note) to your pull request when submitting your fix. This will guide (you and) the contributors who will review your fix to know what is important to check or test for.
+
+- [ ] The important notice and github issue linking has been updated for the pull request template.
+- [ ] I certify that I ran my checklist
```

Comment:
I'm trying to understand why you are changing testing checklist too @tylerjusfly 
The issue didn't mention to do this.

Reply:
Also why is your branch name not using our new naming convention? @tylerjusfly 
Check that Discord thread.

Reply:
> I'm trying to understand why you are changing testing checklist too @tylerjusfly The issue didn't mention to do this.

oh really ? okay i will revert that and push

Reply:
Sorry for the oversight.